source/features/transactions/components/TransactionInfo.tsx

```diff
@@ -0,0 +1,102 @@
+import { observer } from 'mobx-react-lite';
```

Comment:
Noticed this component is much similar to transaction list row component, and we need to find out to make more common component and reuse it in those places.
But I think that would be good to do after merging.

Reply:
Agree. But for now keep it like this until we merge all of our changes.